Resources/Prototypes/Entities/markers/construction_ghost.yml

```diff
@@ -10,7 +10,7 @@
   - type: Collidable
   - type: Clickable
     baseshader: unshaded
-    selectionshader: selection_outline_unshaded
+    selectionshader: unshaded
```

Comment:
Just remove the entire line, actually.

The need for a separate, unshaded outline shader was removed when I made outline use a separate framebuffer. Setting the outline shader to unshaded will just break the outlines.

Reply:
Done, removed line 12 & 13 and outlines now show up properly on the construction ghosts: https://i.imgur.com/YHjtPPt.png